src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -65,13 +66,18 @@ function composeDrillContextGroup(
     points: IHighchartsPointObject[],
     chartType: ChartType,
 ): IDrillEventContextGroup {
+    const isCombo = isComboChart(chartType);
```

Comment:
I'm not happy to see a condition for particular visualization type in this general code. We should definitely avoid that. Think if there's another way. I'll think about it too.

Reply:
I have changed the solution that no need to check visualization type there, just simply add custom props if any series has `type` prop.
Please help to review again :)